app/kuma-cp/cmd/migrate.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"github.com/Kong/kuma/pkg/config"
+	kuma_cp "github.com/Kong/kuma/pkg/config/app/kuma-cp"
+	"github.com/Kong/kuma/pkg/config/core/resources/store"
+	core_plugins "github.com/Kong/kuma/pkg/core/plugins"
+	"github.com/pkg/errors"
+
+	"github.com/spf13/cobra"
+)
+
+var migrateLog = controlPlaneLog.WithName("migrate")
+
+func newMigrateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "migrate",
+		Short: "Migrate database to which Control Plane is connected",
+		Long:  `Migrate database to which Control Plane is connected.`,
+	}
+	cmd.AddCommand(newMigrateUpCmd())
+	return cmd
+}
+
+func newMigrateUpCmd() *cobra.Command {
+	args := struct {
+		configPath string
+	}{}
+	cmd := &cobra.Command{
+		Use:   "up",
+		Short: "Run up migrations",
+		Long:  `Run up migrations.`,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			cfg := kuma_cp.DefaultConfig()
+			err := config.Load(args.configPath, &cfg)
+			if err != nil {
+				migrateLog.Error(err, "could not load the configuration")
+				return err
+			}
+
+			ver, err := migrate(cfg)
+			if err != nil {
+				if err == core_plugins.AlreadyMigrated {
+					cmd.Printf("DB already migrated to the newest version: %d", ver)
+				} else {
+					return err
+				}
+			} else {
+				cmd.Printf("DB migrated to %d version", ver)
```

Comment:
Same considerations as above.

As a user, I wouldn't understand a message that says `1579518998` instead of `0.4.0`